util/logger/logger.go

```diff
@@ -89,14 +93,45 @@ func addFields(event *zerolog.Event, keyvals ...interface{}) *zerolog.Event {
 }
 
 func NewSubLogger(name string, obj fmt.Stringer) *SubLogger {
+	var writers []io.Writer
+	var fl *lumberjack.Logger
+	var logDirectory string
+	var logFilename string
+	maxLogSize := 100
+
+	// console writer
+	writers = append(writers, zerolog.ConsoleWriter{Out: os.Stderr})
+
+	// file writer
+	currentDirectory, err := os.Getwd()
+	if err != nil {
+		goto ConsoleLogger
+	}
+
+	logDirectory = filepath.Join(currentDirectory, LogDirectory)
+	logFilename = filepath.Join(logDirectory, LogFilename)
+
+	if err = os.MkdirAll(logDirectory, 0o744); err != nil {
```

Comment:
Based on [lumberjack](https://github.com/natefinch/lumberjack/blob/v2.0/README.md#type-logger) backup logs will be saved in the same directory. We don't need to create a directory for that.